model/src/main/kotlin/PackageReference.kt

```diff
@@ -23,6 +23,7 @@ import com.fasterxml.jackson.annotation.JsonAlias
 import com.fasterxml.jackson.annotation.JsonInclude
 
 import java.util.SortedSet
+import java.util.Stack
```

Comment:
It would be cool if the commit message explained on a high level how the new algorithm works, and how it achieves the performance gain in particular.

Reply:
Done.